multicore_runtime/cloud_logging_test.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import json
+import logging
+import mock
+import os
+import unittest
+
+import cloud_logging
```

Comment:
I wonder if we should be using explicitly relative imports here...

```python
  import .cloud_logging
```

Do you have a reason to rely on the relative import working this way here?

(I admit I don't really have a lot of experience with this feature given I mostly work with code that always uses absolute imports for all module names, so if you know or can find some of the reasons why to use it or not to use it I'd appreciate it. I can also pull in others to weigh in on the possible approaches.)